client-templates/azure-repos/accept.json.sample

```diff
@@ -0,0 +1,222 @@
+{
+  "public": [
+    {
+      "//": "used for pushing up webhooks from Azure",
+      "method": "POST",
+      "path": "/webhook/azure-repos/webhookId"
+    }
+  ],
+  "private": [
+    {
+      "//": "get list of projects for given organisation",
+      "method": "GET",
+      "path": "/_apis/projects",
+      "origin": "https://${AZURE_REPOS_HOST}/${AZUR_REPOS_ORG}",
+      "auth": {
+        "scheme": "basic",
+        "token": "${BROKER_CLIENT_VALIDATION_BASIC_AUTH}"
```

Comment:
Could do 
```suggestion
        "username": "PAT",
        "password": "${AZURE_REPOS_TOKEN}"
```

This way the basic auth header would work without any change

Reply:
Yes, you are right. But it seems for me a bit confusing. `PAT` is defiantly not a username. I prefer to find better way

Reply:
I agree with @AlbertKogan here, let's keep it explicit rather than hacking around the existing logic.